cmake_modules/commonSetup.cmake

```diff
@@ -126,6 +126,10 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   option(WSSQL_SERVICE "Configure use of ws_sql service" ON)
   option(USE_DIGISIGN "Use digisign" ON)
   set(CUSTOM_PACKAGE_SUFFIX "" CACHE STRING "Custom package suffix to differentiate development builds")
+  option(USE_PROXIES  "Install default eclmod files" ON)
```

Comment:
I don't think this is quite what is needed - I don't think we want to configure the target directory.
@GordonSmith I think we need your input.
What we want is for the installer for the client tools to have an option whether to install the proxies from the eeproxies directory to the plugins directory.  I think you probably want (the same?) option which controls whether the plugins are installed as part of make install.

Reply:
@ghalliday Is there any reason to not always install them?

Reply:
If it wasn't for lib_date, then I would say no reason not to.  @richardkchapman maybe we should say that lib_date is not supported in 7.8.0 and following.

Reply:
Include it and embed some deprecated warnings for 7.8?

Reply:
It is ancient code, which is unsupported, and will potentially pollute the global name space.  I would vote for it never seeing the light of day.

Reply:
In which case, don't include it and add some instructions to the red book to manually add it if really needed?

Reply:
It may well already exist in the gitlab repo.  I'll try and check.

Reply:
Let see if I understand correctly:
1) I still keep old proxies under plugins/proxies and new ones  under plugins/eeproxies/
2) For Platform build both old and new proxies will be installed to plugins/
3) For Clienttools old proxies will be installed to plugins and USE_EEPROXIES will decide if install new proxies (eeproxies)
4) not include lib_date in platform plugins/

Reply:
It looks like there are a  reasonable number of references to lib_date in the exiting ECL code base.

Reply:
OK. I will move it to platform plugins or plugins/eeproxies/